src/SBVonKarman.cpp

```diff
@@ -125,14 +126,15 @@ namespace galsim {
 
     // Note: lam and L0 are both in units of r0, so are dimensionless within VKInfo.
     VonKarmanInfo::VonKarmanInfo(double lam, double L0, bool doDelta,
-                                 const GSParamsPtr& gsparams) :
+                                 const GSParamsPtr& gsparams, double force_stepk) :
         _lam(lam), _L0(L0),
         _L0_invcuberoot(fast_pow(_L0, -1./3)), _L053(fast_pow(L0, 5./3)),
+        _stepk(0.0), _maxk(0.0),
         _delta(exp(-0.5*magic1*_L053)),
         _deltaScale(1./(1.-_delta)),
         _lam_arcsec(_lam * ARCSEC2RAD / (2.*M_PI)),
         _doDelta(doDelta), _gsparams(gsparams),
-        _radial(Table::spline)
+        _radial(Table::spline), _sampler(nullptr)
```

Comment:
`nullptr` is a C++11 thing.  While probably almost all users will have that now, we technically don't require C++11 with SCons installations (using Boost rather than pybind11).
So this should be just `_sampler()` instead.  Or indeed just leaving it off is actually fine.